src/Sylius/Component/Core/Model/AdjustmentInterface.php

```diff
@@ -16,8 +16,10 @@
 interface AdjustmentInterface extends BaseAdjustmentInterface
 {
     // Labels for tax, shipping and promotion adjustments.
-    const TAX_ADJUSTMENT = 'tax';
-    const SHIPPING_ADJUSTMENT = 'shipping';
-    const ORDER_PROMOTION_ADJUSTMENT = 'order_promotion';
     const ORDER_ITEM_PROMOTION_ADJUSTMENT = 'order_item_promotion';
+    const ORDER_PROMOTION_ADJUSTMENT = 'order_promotion';
+    const ORDER_SHIPPING_PROMOTION_ADJUSTMENT = 'order_shipping_promotion';
```

Comment:
We could forget about trying to bring over the `Order::getShippingTotal` method from Sylius to avoid confusion.

What if someone accidentally uses `ORDER_SHIPPING_PROMOTION_ADJUSTMENT` in the future in our code, when they shouldn't?